docs/source/user-guide/sql/information_schema.md

```diff
@@ -22,7 +22,10 @@
 DataFusion supports showing metadata about the tables and views available. This information can be accessed using the
 views of the ISO SQL `information_schema` schema or the DataFusion specific `SHOW TABLES` and `SHOW COLUMNS` commands.
 
-To show tables in the DataFusion catalog, use the `SHOW TABLES` command or the `information_schema.tables` view:
+## `SHOW TABLES`
```

Comment:
This makes the table of contents have headers so it is easier to find these items I think

<img width="990" alt="Screenshot 2024-12-20 at 9 49 56 PM" src="https://github.com/user-attachments/assets/44451c61-c4c9-42c9-9f9a-07c984a7e2cd" />